pghrep/src/main.go

```diff
@@ -0,0 +1,248 @@
+/* 2018 © Dmitry Udalov dmius@postgres.ai
+ 2018 © Postgres.ai
+
+Perform a generation *md reports on base of results health checks
+Usage: 
+pghrep --checkid=XXX --checkdata=file:///path_to_check_results.json --outdir=/home/results
+*/
+package main
+
+import (
+	"fmt"
+    "os"
```

Comment:
the previous line is 1 tab
this one is 4 spaces

in Go programs, the most popular approach is 1 tab (usually visualized as 8 spaces).

let's decide, what will we use?
/cc @Nastradamus

Reply:
We can also use `go fmt` in githook.